docs/docs/actions.mdx

```diff
@@ -2,6 +2,24 @@
 id: actions
 sidebar_label: Overview
 title: Actions
+abstract: After each user message, the dialogue manager will predict an action that the assistant should perform next. This page gives you an overview of the different types of actions you can use.
```

Comment:
Do we ever talk about "the dialogue manager" as something that predicts stuff? I think we say "the model" or "the dialogue management model" or something

not against it, just don't think we use it anywhere else